### PR TITLE
Use `subwasm` to export chain metadata #NX-30

### PR DESCRIPTION
**Pull request by obsessed-cake**

### CODE REVIEW

**Comment by ComposableFiB0t** [comment]
Task linked: [NX-30 Use `subwasm` to export chain metadata](https://app.clickup.com/t/20465559/NX-30)

File: subwasm.nix [hunk 1]
```diff
@@ -43,6 +43,32 @@
...
       };
+
+      mkChainMetadata = { runtime, chainName }:
+        pkgs.stdenv.mkDerivation {
+          name = "subwasm-get-${chainName}-metadata";
+          dontUnpack = true;
+
+          installPhase = ''
+            mkdir $out
+            ${subwasm}/bin/subwasm metadata --json ${runtime}/lib/runtime.optimized.wasm > $out/${chainName}-metadata.json;
```
Comment (dzmitry-lahoda) [inline subwasm.nix:54]: use getExe to get path to subwasm

**Review by dzmitry-lahoda — APPROVED** [review]

File: subwasm.nix (hunk 1 above)
Comment (obsessed-cake) [reply subwasm.nix]: fixed.

File: .github/workflows/build-test-deploy.yml [hunk 2]
```diff
@@ -778,6 +778,25 @@ jobs:
...
+    concurrency:
+      group: ${{ github.workflow }}-package-dali-metadata-${{ github.ref }}
+      cancel-in-progress: true
+    container:
+      image: niteo/nixpkgs-nixos-22.05:316b762afdb9e142a803f29c49a88b4a47db80ee
+    steps:
+      - uses: actions/checkout@v3
+      - uses: "./.github/templates/watch-exec"
+        with:
+          command: nix -- build .#subwasm-get-dali-metadata
```
Comment (KaiserKarel) [inline .github/workflows/build-test-deploy.yml:797]: should we build for picasso and composable too?

File: .github/workflows/build-test-deploy.yml [hunk 3]
```diff
@@ -778,6 +778,25 @@ jobs:
           command: nix -- build .#price-feed
           token: "${{ secrets.CACHIX_AUTH_TOKEN }}"
 
+  package-dali-metadata:
```
Comment (KaiserKarel) [inline .github/workflows/build-test-deploy.yml:781]: Should add this flow to effects-gate too

File: .github/workflows/build-test-deploy.yml (hunk 2 above)
Comment (obsessed-cake) [reply .github/workflows/build-test-deploy.yml:797]: I think it will be useful. Let's add it.

Comment (dzmitry-lahoda) [reply .github/workflows/build-test-deploy.yml:797]: this one was next task in cu. we tried to end to end floq for one client and then wnable generation for all.

Comment (dzmitry-lahoda) [reply .github/workflows/build-test-deploy.yml:797]: dali is prio for bridge.

Comment (dzmitry-lahoda) [reply .github/workflows/build-test-deploy.yml:797]: and it is not clear should we buld historical clienta or not.

Comment (obsessed-cake) [reply .github/workflows/build-test-deploy.yml:797]: this doesn't require a lot of places, JSON with metadata is less than 1.5 Mb.
I added a packaging of picasso and composable too.

File: .github/workflows/build-test-deploy.yml (hunk 3 above)
Comment (obsessed-cake) [reply .github/workflows/build-test-deploy.yml:781]: added.

**Review by dzmitry-lahoda — APPROVED** [review]

**Comment by obsessed-cake** [comment]
This is duplicate of https://github.com/ComposableFi/composable/pull/2677

**Pull request closed without merging**